eth_tester/backends/pyethereum/v20/main.py

```diff
@@ -376,6 +379,19 @@ def get_transaction_receipt(self, transaction_hash):
             transaction_hash,
         )
         state = _get_state_by_block_hash(self.evm, block.hash)
+
+        if self.get_fork_block('FORK_BYZANTIUM') is not None:
```

Comment:
Shouldn't this be compared to the current block number?  In a chain where Byzantium is enabled but not yet active, this would try to push a `status` field into the receipt.